libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -101,6 +101,7 @@
         OPENED_MEDIA_LIBRARY,
         OPENED_BLOG_SETTINGS,
         OPENED_ACCOUNT_SETTINGS, // Future
```

Comment:
You can remove the "//future" comment on all of these events, since they're already tracked in Trackers.